admin/src/react-components/accounts.js

```diff
@@ -168,10 +169,10 @@ export const AccountList = withStyles(styles)(
         }, 6000);
       }
       render() {
-        const { classes } = this.props;
+        const { classes, refreshView, ...other } = this.props;
```

Comment:
we are no longer passing through `refreshView` here. Is that intended?

Reply:
Yes, thanks for checking, it's being used where it needs to be in the function above outside of render().

Reply:
Added comment to make this clearer: // Do not pass refreshView to other components, only onCreateAccounts()

Was throwing an error getting passed in.

Reply:
Added eslint ignore line as well.